esp/logging/logginglib/logthread.cpp

```diff
@@ -157,8 +157,7 @@ bool CLogThread::queueLog(IEspUpdateLogRequestWrap* logRequest)
 {
     unsigned startTime = (getEspLogLevel()>=LogNormal) ? msTick() : 0;
```

Comment:
after looking at msTick, it does seem to have the potential to be relatively expensive, almost certainly more expensive than the getEspLogLevel... 
I suppose my original question should have been more directed towards the fact we might save some cycles/io at the expense of potentially having a misleading variable name.
I'd feel better if the var name somehow conveyed the fact that it's only the startTime IF loglevel is set to a particular setting...